Daf.Core.Ssis.Wrapper/AssemblyLoader.cs

```diff
@@ -106,8 +106,8 @@ public static int VersionNumber
 			}
 			set
 			{
-				if (versionNumber.HasValue)
-					throw new NotSupportedException("Changing the AssemblyLoader version after it has already been set is not supported.");
+				//if (versionNumber.HasValue)
+				//	throw new NotSupportedException("Changing the AssemblyLoader version after it has already been set is not supported.");
```

Comment:
This should probably not just be commented away, but I'm not sure how to handle this properly. Any input?

Reply:
Let's discuss it!

Reply:
After discussion:

if same version do nothing, else throw notimplementedexception "don't support changing version".

TODO: Unload all loaded assemblies on versionNumber update